### DISCUSSION TITLE
How to select the `Text`?

### DISCUSSION THREAD

**Discussion by andrew-ares** [bdlukaa/fluent_ui#951] [Q&A] [answered]
In the material library, we have `SelectableText`. Does fluent_ui provide an equivalent alternative?

**Comment by bdlukaa** [comment, marked as answer]
We export material's `SelectableText`. In fact, we even use it in the [example app](https://github.com/bdlukaa/fluent_ui/blob/master/example/lib/screens/surface/expander.dart#L137-L149). 

There isn't much to do since a selectable text is not a native control. Tho, there is https://github.com/flutter/flutter/issues/99231, which is an issue with Flutter and would improve the look and feel for selectable texts. The behavior I described in the issue is the default for Windows.

There may be the possibility to map material's `SelectableText` to use the `FluentThemeData.accentColor` by default, but that seems such an overhead.

**Reply by andrew-ares** [reply]
Thanks for the help!
